x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -28,22 +28,31 @@ import { useTestIdGenerator } from '../../../../hooks/use_test_id_generator';
 import { useDataTestSubj } from '../../hooks/state_selectors/use_data_test_subj';
 
 const CommandInputContainer = styled.div`
+  background-color: ${({ theme: { eui } }) => eui.euiColorGhost};
   background-color: ${({ theme: { eui } }) => eui.euiFormBackgroundColor};
+  border-radius: ${({ theme: { eui } }) => eui.euiBorderRadius};
+  outline: solid ${({ theme: { eui } }) => eui.euiBorderWidthThin}
```

Comment:
would using `eui.euiBorderThin` work better here? as in:

```suggestion
  outline: ${({ theme: { eui }}) => eui.euiBorderThin}
```

(Its defined as `"euiBorderThin": "1px solid #d3dae6",`)